OWNERS_ALIASES

```diff
@@ -23,8 +21,10 @@ aliases:
   # folks who may have context on ancient history,
   # but are no longer directly involved
   kubebuilder-emeritus-approvers:
+  - adirio
   - directxman12
   - droot
-  - mengqiy
   - estroz
-  - adirio
+  - jmrodri
+  - mengqiy
+  - pwittrock
```

Comment:
Awe this makes me sad :( but it makes sense. +1

Reply:
It makes me too sad. Came back?